packages/react-native-ui-lib/src/components/progressBar/progressBar.api.json

```diff
@@ -29,6 +29,12 @@
       "name": "customElement",
       "type": "JSX.Element",
       "description": "Custom element to render on top of the animated progress"
+    },
+    {
+      "name": "borderRadius",
+      "type": "number",
+      "description": "Custom border radius for the progress bar",
+      "default": "BorderRadiuses.br100"
```

Comment:
This is not accurate, the default is more complex (depends if it's full width or not), I think we should remove it